src/lib/y2storage/device_description.rb

```diff
@@ -176,11 +220,21 @@ def lvm_pv_type_label(lvm_pv)
     def snapshot_type_label(lvm_snapshot)
       label =
         if lvm_snapshot.is?(:lvm_thin_snapshot)
-          # TRANSLATORS: %{origin} is replaced by an LVM logical volumme name
-          # (e.g., /dev/vg0/user-data)
-          _("Thin Snapshot of %{origin}")
+          if lvm_snapshot.encrypted? && include_encryption
+            # TRANSLATORS: %{origin} is replaced by an LVM logical volume name
+            # (e.g., /dev/vg0/user-data)
+            _("Encrypted thin Snapshot of %{origin}")
```

Comment:
I am not sure about using capital letter for "Snapshots". If so, I would use it for both Thin and Snapshots.